Taller152/src/paqutedos/Principal.java

```diff
@@ -5,11 +5,47 @@
  */
 package paqutedos;
 
+import java.util.Scanner;
 
 /**
  *
  * @author reroes
  */
 public class Principal {
-    
+
+    public static void main(String[] args) {
+        Scanner entrada = new Scanner(System.in);
+        boolean bandera = true;
+        String nombreEquipo;
+        double presupuestoAnual;
+        String nombreEstadio;
+        String cadenaAcumuladora = "";
+        String opcion;
+        try {
+            do {
+                System.out.println("Nombre del equipo de baloncesto: ");
+                nombreEquipo = entrada.nextLine();
+                System.out.println("Ingresar el presupuesto anual: ");
+                presupuestoAnual = entrada.nextDouble();
+                entrada.nextLine();
+                System.out.println("Ingresar el nombre del estadio: ");
+                nombreEstadio = entrada.nextLine();
+                cadenaAcumuladora = String.format("%s%s, presupuesto %.1f; "
+                        + "estadio %s\n", cadenaAcumuladora, nombreEquipo,
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.